parcels/particle.py

```diff
@@ -36,6 +37,57 @@ def AdvectionEE(particle, grid, time, dt):
     particle.lat += v1 * dt
 
 
+def AdvectionRK45(particle, grid, time, dt):
+    tol = [1e-9]
```

Comment:
Shouldn't this `tol` be set by the user? It is a keyword in the `.execute` now, but doesn't seem to be passed on? (Thanks @Abobie for pointing out!)